lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
This is due to the fact that the original (pre-gem) pub-sub implementation set the "raw_delivery" flag to true.

For compatibility, we should do the same for the new implementation, because otherwise subscriptions made by the old one will be incompatible with the new one.